app/controllers/course_clone_controller.rb

```diff
@@ -29,4 +31,8 @@ def check_permission
   def clone_params
     params.permit(:campaign_slug)
   end
+
+  def clone_assignments_params
+    params.permit(:copy_assignments)
```

Comment:
Since this is also a clone param, and you're digging into it for the `:copy_assignments` value anyway, I think this should be done by extending `clone_params` to permit the additional value, rather than using a separate method.